Seminars/Sem.02/Pract.02/Task06.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <fstream>
+
+const int BUFFER_SIZE = 256;
+
+int main()
+{
+	std::ifstream myFileToRead("Task06.cpp");
+
+	if (!myFileToRead.is_open()) {
+		std::cout << "Error!";
+		return -1;
```

Comment:
The whole code should be in a separate function + avoid return -1 for error code